website/docs/tutorials/06-deploying/02-migrations/invoke.md

```diff
@@ -16,10 +16,9 @@ This cheatcode invokes a StarkNet contract, with possible state changes. Can be
 It's an extension of [CheatcodeNetworkConfig](../03-network-config.md), so it's properties are applicable here as well.
 
 ```python
-Wei = int
+from protostar.starknet_gateway import Fee
```

Comment:
```suggestion
Wei = int
Fee = "auto" | Wei
```

Reply:
[fix type in docs](https://github.com/software-mansion/protostar/pull/979/commits/701c140ae19de3937f43a7dd8828e489cff4de5b)